.github/workflows/e2e.yml

```diff
@@ -0,0 +1,122 @@
+name: e2e
+
+on: [push]
+env:
+  REGISTRY: 'ghcr.io/dell/csi-baremetal'
+  go_version: '1.16.8'
+  golangci_version: '1.37.0'
+  helm_version: '3.7.1'
+  kubectl_version: 'v1.22.2'
+jobs:
+  e2e:
+    runs-on: ubuntu-20.04
+    if: "contains(github.event.head_commit.message, '[e2e]')"
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      
+    - name: Add csi-baremetal-operator
+      uses: actions/checkout@v2
+      with:
+        repository: dell/csi-baremetal-operator 
+        path: ./csi-baremetal-operator 
+
+    - name: Set environment variables
+      run: |
+        echo "CSI_BAREMETAL_DIR=$RUNNER_WORKSPACE/csi-baremetal/" >> $GITHUB_ENV
+        echo "CSI_BAREMETAL_OPERATOR_DIR=$RUNNER_WORKSPACE/csi-baremetal/csi-baremetal-operator/" >> $GITHUB_ENV
+
+    - name: LVM2 install
+      run: sudo apt-get install -y lvm2
+
+    - name: Prepare Golang 
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{env.go_version}}
+
+    - name: Install helm
+      uses: azure/setup-helm@v1
+      with:
+        version: ${{env.helm_version}}
+          
+    - name: Kubectl install
+      uses: azure/setup-kubectl@v1
+      with:
+        version: ${{env.kubectl_version}} 
+
+    - name: add CSI_VERSION
+      run: |
+        cd ${{env.CSI_BAREMETAL_DIR}}
+        echo "CSI_VERSION=`make version`" >> $GITHUB_ENV
+
+    - name: Get dependencies
+      run: make dependency
+
+    - name: Generate CRD
+      run: |
+        make install-controller-gen 
+        make generate-deepcopy
+
+    - name: Build binary
+      run: |
+        make build
+        make DRIVE_MANAGER_TYPE=loopbackmgr build-drivemgr
+
+    - name: Download health-probe
+      run: make download-grpc-health-probe
+
+    - name: Build docker images
+      run: |
+        make images REGISTRY=${{ env.REGISTRY }}
+        make DRIVE_MANAGER_TYPE=loopbackmgr image-drivemgr REGISTRY=${{ env.REGISTRY }}
+    
+    - name: docker-build operators 
+      run: |
+        cd ${{env.CSI_BAREMETAL_OPERATOR_DIR}}
+        echo "CSI_OPERATOR_VERSION=`make version`" >> $GITHUB_ENV
+        make docker-build REGISTRY=${{ env.REGISTRY }}
+
+    - name: Kind preparation
+      run: |
+        cd ${{env.CSI_BAREMETAL_DIR}}
+        make kind-build KIND_VER=0.11.1
+        make kind-create-cluster-v19
+        kubectl cluster-info --context kind-kind
+        kubectl get pods -o wide --all-namespaces 
+        echo "current-context:" $(kubectl config current-context)
+        echo "environment-kubeconfig:" ${KUBECONFIG}
+      
+    - name: Check pods
```

Comment:
Can we use `https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster`  `--wait 30s` in Kind preparation ?

Reply:
+1

Reply:
done